lib/Frontend/Frontend.cpp

```diff
@@ -1192,6 +1192,9 @@ ImplicitImportInfo CompilerInstance::getImplicitImportInfo() const {
     pushImport(CXX_SHIM_NAME, {ImportFlags::ImplementationOnly});
   }
 
+  if (Invocation.getClangImporterOptions().BridgingHeaderAsModule)
+    pushImport(CLANG_HEADER_MODULE_NAME, {ImportFlags::ImplementationOnly});
```

Comment:
Note: We can do `ImplementationOnly` for resilient modules, and `Exported` for others. In this case, we can support bridging header for frameworks that uses library evolution.